xapian-core/diversify/diversify.cc

```diff
@@ -146,20 +133,19 @@ Diversify::Internal::compute_diff_dmset(const vector<Xapian::docid>& dmset)
 }
 
 double
-Diversify::Internal::evaluate_dmset(const vector<Xapian::docid>& dmset)
+Diversify::Internal::evaluate_dmset(const vector<Xapian::docid>& dmset,
+			  const Xapian::ClusterSet& cset)
```

Comment:
The continuation line looks underindented to me.

Reply:
Correction pushed in a fixup commit.